bee-message/src/error.rs

```diff
@@ -73,8 +73,8 @@ impl fmt::Display for Error {
         match self {
             Error::CryptoError(e) => write!(f, "Cryptographic error: {}.", e),
             Error::DuplicateUtxo(utxo) => write!(f, "Duplicate UTXO {:?} in inputs.", utxo),
-            Error::DuplicateSignature(index) => {
-                write!(f, "Duplicate signature at index: {0}", index)
+            Error::DuplicateSignatureUnlockBlock(index) => {
+                write!(f, "Duplicate signature unlock block at index: {0}", index)
```

Comment:
ThEsE sHoUlD bE lOwErCaSe!!!1!!one!!eleven!!11!.

Reply:
Will change :)